webapp/components/admin_console/session_settings.jsx

```diff
@@ -54,7 +54,7 @@ export default class SessionSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.webSessionDays'
-                            defaultMessage='Session Length for Web in Days:'
+                            defaultMessage='Web Session Length (Days):'
```

Comment:
Talked to Corey. Web basically refers to email and LDAP. SSO refers to gitlab auth. UI'm updating the help text and then making a ticket to combine these 2 fields into one moving forward.